1.8/development/create-package.md

```diff
@@ -0,0 +1,153 @@
+---
+post_title: Creating a Universe Package
+nav_title: Creating a Universe Package
+menu_order: 10
+---
+
+This page covers general advice and information about creating a DC/OS Universe package. Consult the [Publish a Package][2] page of the Universe documentation for full details.
+
+Each DC/OS Universe package consists of 4 JSON files:
+
+* `package.json` - High-level metadata about the package.
+* `resource.json` - Contains all of the externally hosted resources (e.g. Docker images, HTTP objects and images) that are required to install the application.
+* `config.json` - Configuration properties supported by the package, represented as a json-schema.
+* `marathon.json.mustache` - A mustache template that, when rendered, creates a Marathon app definition capable of running your service.
+
+# `package.json`
+
+Every package in Universe must have a `package.json` file that specifies the highest-level metadata about the package (comparable to a package.json in Node.js or setup.py in Python).
+
+Currently, a package can specify one of two values for `.packagingVersion`, either 2.0 or 3.0. The version declared will dictate which other files are required for the complete package as well as the schemas all the files must adhere to.
+
+Consider the following guidelines when creating a your `package.json` file:
```

Comment:
Confusing: `a your`

I imagine it should be either "when creating a `package.json` file" or "when creating your `package.json` file"